Tests/TuistCoreTests/Graph/GraphTraversingTests.swift

```diff
@@ -0,0 +1,225 @@
+import Mockable
+import Path
+import TuistSupportTesting
+import XcodeGraph
+import XCTest
+@testable import TuistCore
+
+final class GraphTraversingTests: TuistUnitTestCase {
+    func test_filterIncludedTargets_when_graph_is_empty() {
+        // Given
+        let subject = MockGraphTraversing()
```

Comment:
This might be controversial... the subject is a mock because the actual code to be tested is a protocol extension. This makes it easy to configure the given.

Reply:
Does that mean that an instance is not necessary at all? If so, what do you think if we turn the function into `static`? I think that'd be more appropriate and eliminate the confusion here.